03_08_2018/bug-hunt/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply plugin: 'com.android.application'
```

Comment:
maybe rename folder to have year first, then month then day so that it will be alphabetically sortable?